spec/02-integration/05-proxy/01-router_spec.lua

```diff
@@ -303,21 +302,23 @@ describe("Router", function()
       insert_apis {
         {
           name         = "api-strip-uri",
-          upstream_url = "http://httpbin.org",
+          upstream_url = helpers.mock_upstream_url,
           uris         = { "/x/y/z", "/z/y/x" },
           strip_uri    = true,
         },
       }
 
       helpers.run_migrations()
-      assert(helpers.start_kong())
+      assert(helpers.start_kong({
+        nginx_conf = "spec/fixtures/custom_nginx.template",
```

Comment:
This I could not do easily. The kong_exec facility is used in other places (for example the cmd). I could create a `kong_test_exect` command to isolate from this, but at that point that's not hugely different than just adding a param. I've left it as it is.